### DISCUSSION TITLE
Android App upload or Backup

### DISCUSSION THREAD

**Discussion by 849025143** [immich-app/immich#12859] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Android App add Upload, not only Backup.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Your feature request is way too vague to actually tell what you're asking for, but uploading individual assets manually is already supported.
